medical_prescription_state/views/medical_prescription_order_line.xml

```diff
@@ -42,6 +42,20 @@
         </field>
     </record>
 
+    <record id="medical_prescription_order_line_form_view_stages" model="ir.ui.view">
+        <field name="name">medical.prescription.order.line.form</field>
+        <field name="model">medical.prescription.order.line</field>
+        <field name="inherit_id" ref="medical_prescription.medical_prescription_order_line_view_form" />
+        <field name="mode">primary</field>
```

Comment:
TIL setting the view mode to primary is used when you want to inherit a view but not override that original view. For example: any views inheriting from [here](https://github.com/LasLabs/vertical-medical/blob/release/10.0/medical/views/medical_abstract_entity.xml) will want to use mode=primary because we do not want to override that view, but just make a copy of it. Medical Patient view does this, for example.

In this case, I believe we want to override the medical_prescription_order_line_view so this line should be removed.